python/shotgun_model/shotgun_hierarchy_model.py

```diff
@@ -961,16 +938,22 @@ def hierarchy_is_supported(sg_connection):
 
     :param sg_connection: A shotgun connection.
 
-    :returns: ``True`` if hierarchy supported, ``False`` otherwise.
+    :returns: A tuple of 2 items where the first item is a boolean indicating
+        whether hierarchy is supported. If hierarchy is supported, the second
+        item will be ``None``. If hierarchy is not supported, the second item
+        will be a string explaining why.
+
     """
 
     server_caps = sg_connection.server_caps
 
     # make sure we're greater than or equal to SG v7.0.2
-    return (
-        hasattr(sg_connection, "server_caps") and
-        server_caps.version and
-        server_caps.version >= (7, 0, 2) and
-        hasattr(sg_connection, "nav_expand")
-    )
+    if not (hasattr(sg_connection, "server_caps") and
+            server_caps.version and
+            server_caps.version >= (7, 0, 2)):
+        return (False, "The version of SG being used does not support hierarchy queries. v7.0.2 is required.")
```

Comment:
Maybe it's because I'm not a native english speaker, but I think "does not support querying for the hierarchy"